src/Xamarin.Android.Tools.AndroidSdk/Xamarin.Android.Tools.AndroidSdk.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$title$</title>
+    <authors>$author$</authors>
```

Comment:
How does this work/where does `$author$` come from?  I don't see any other relevant substring matches for `author` in this PR.

Ditto the other elements within this `.nuspec` file.

Reply:
These tokens are placeholders for values contained in `AssemblyInfo.cs`. `$author$` maps to `AssemblyCompany`.